.github/workflows/publish.yml

```diff
@@ -15,25 +15,12 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
       - name: Install dependencies
-        run: npm install
-      - name: Build puppeteer
-        run: npm run build
-      - name: Publish puppeteer
-        env:
-          NPM_TOKEN: ${{secrets.NPM_TOKEN_PUPPETEER}}
         run: |
-          npm config set registry 'https://wombat-dressing-room.appspot.com/'
-          npm config set '//wombat-dressing-room.appspot.com/:_authToken' '${NPM_TOKEN}'
-          echo "Publishing puppeteer"
-          npm publish
-          # DEPRECATED_RANGE=$(node utils/get_deprecated_version_range.js)
-          # echo "Deprecating old puppeteer versions: $DEPRECATED_RANGE"
-          # npm deprecate puppeteer@$DEPRECATED_RANGE "Version no longer supported. Upgrade to @latest"
-      - name: Publish puppeteer-core
+          npm install
+          npx lerna bootstrap
+      - name: Build packages
+        run: npm run build
+      - name: Publish packages
         env:
-          NPM_TOKEN: ${{secrets.NPM_TOKEN_PUPPETEER_CORE}}
-        run: |
-          utils/prepare_puppeteer_core.js
-          npm config set registry 'https://wombat-dressing-room.appspot.com/'
-          npm config set '//wombat-dressing-room.appspot.com/:_authToken' '${NPM_TOKEN}'
-          npm publish
+          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN_RELEASE}}
```

Comment:
so will it work with 2FA?

Reply:
Yes, the packages have the repository hardcoded in their package.json

Reply:
Does hardcoding the proxy as a registry name in the package.json affect anything for the end-users (e.g., during npm install etc)?

Reply:
Found the docs 📝

Reply:
No, it's just for development